### ISSUE TITLE
Open DjangoCon Europe Support Group membership to aspiring organisers. 

### ISSUE THREAD

**Issue by carltongibson** [django/dsf-working-groups#1]
I know it's draft, so suggesting a tweak. 

DjangoCon EU Support Group membership [currently states](https://github.com/django/dsf-working-groups/blob/main/active/dceu.md#future-membership):

> Membership is open to former organizers of DjangoCon EUs. 

That makes sense. 

**BUUUT**... there are lots of people at each DjangoCon EU that have a _Ooo, maybe I could do this near me_ moment. They should be able to join (maybe as an _associate member_ or something) so that they can get a taste, have access to resources on _How to do it…_, and so on. 

It might take a year or two for that moment to grow into an actual drive to do it, but we want to capture and nurture that moment (where at the moment it kind of just evaporates...)

**Comment by rixx** [comment]
I'm not sure I agree (no surprise there, I wrote and submitted the initial draft for the team). 

First off, I definitely agree that hopeful/future/interested organisers should be able to get experience and insight into event running *before* taking on DCEU. Not being able to do that is a big part of the organiser burnout we're seeing imo. That's actually part of the team's responsibilites: putting future organisers in touch with the current ones, so that they can join the team / shadow the organisers and learn what the job involves etc. Imo that's the best way to get more experienced, and this team should make that a possibility as much as possible (by increasing the lead time on the choice of team/venue, making this option to join the current team a well-known option etc).

However, this team's responsibilities were chosen to fit people who already have experience running DCEU: Vetting proposals by potential organisers (including sanity-checking the budget), providing regular check-ins if wanted to make sure nothing is forgotten, and collecting all those resources that make running an event easier and better. Part of them will be published to be accessible for everybody, especially prospective organisers, while another part would be available on request (past budgets, sponsor contacts etc, things you don't want to put on a public website).

**Comment by carltongibson** [comment]
Yep, I see the sense here! 

The thing I've felt missing is some explicit **reach out here if you have this feeling**, so that interested (however slightly) have a clear way to grow that. 

(The callout at various DjangoCons has been "If you want to do it", which might be a bit daunting. Something a wee-bit lighter…)

Happy if you want to close this if that's _already considered_. 

Thanks @rixx!

**Comment by carltongibson** [comment]
> We’ve not received any replies ... 

@thibaudcolas Grrr. Just as a data point, I was (still am) pondering whether I could do this one year. I didn't have capacity this year, but I looked a the post and the licensing agreement linked there, and was just overwhelmed: it's all obligations, and very little/nothing in the line of "we can help you get this going".

**Comment by thibaudcolas** [comment]
@carltongibson yes, my understanding is this working group is the most tangible kind of help facilitated by the DSF, and the call for proposal and licensing agreement predate the working group being up and running. Aside from the WG, the DSF board’s way of helping organizers is primarily with funding. Looking at [past DSF funding](https://docs.google.com/spreadsheets/d/1XxZEumqNiz1lPUuiPXnx0dXjwpp9HrXdvfFpex9hKRA/edit#gid=1599837429), I believe it’s on the order of $5-10k per event.

Aside from that – it’s up to existing working groups, and individual board members / board decisions, to decide what kind of help is offered. So for example I’m sure the social media WG would be up for helping with conference comms, but it’s also not been active long enough to help with the call for proposal.

---

Anyway, for now @benjaoming and a few others have been working on an updated call for proposal, which will make it quite a bit clearer what kind of help is available.

**Comment by thibaudcolas** [comment]
I believe this is now "solved" as of #46, a new group which is available to a broader group of participants. Coming back to the original feedback:

> there are lots of people at each DjangoCon EU that have a Ooo, maybe I could do this near me moment. They should be able to join (maybe as an associate member or something) so that they can get a taste, have access to resources on How to do it…, and so on.

Here are the relevant responsibilities of the new WG:

> Assist DjangoCon Europe organizers with event planning, logistics, and execution.

And for all events, _Maintain shared resources to facilitate continuity and knowledge transfer between events._ The goal of those resources is to be publicly available and be relevant for all kinds of events.
